examples/nuxt-app/app.vue

```diff
@@ -20,6 +20,8 @@
         </p>
 
         <SwTextField label="A text field" v-model="text" />
+
+        <MtTabs :items="[{ label: 'Foo' }, { label: 'Bar' }]" />
```

Comment:
Though this is enough to test that the DeviceHelper is SSR compatible, I suggest to make this component do something like switching between content. This way we can be sure that switching views also works when using SSR. Additionally it creates a nicer demo page.